src/main/resources/hudson/security/GlobalMatrixAuthorizationStrategy/config.jelly

```diff
@@ -60,7 +60,7 @@ THE SOFTWARE.
             </j:if>
           </j:forEach>
         </j:forEach>
-        <l:isAdmin>
+        <l:hasPermission permission="${instance.getEditingPermission()}">
```

Comment:
I think you could use instance.editingPermission?

Reply:
🤷 no difference there, and makes grepping slightly more straightforward.